File: MORM.CSharp/Classes/Parametro.cs
namespace MORM.Reppositorio.Classes
{
    public class Parametro
    {
        public Parametro(string database, string username, string password, string hostname)
        {
            Database = database;
            Username = username;
            Password = password;
            Hostname = hostname;
        }
        
        public string Database { get; private set; }
        public string Username { get; private set; }
        public string Password { get; private set; }
        public string Hostname { get; private set; }
    }
}

File: MORM.Java/Classes/Parametro.java
package MORM.Java.Classes;

public class Parametro
{
    public Parametro(String database, String username, String password, String hostname)
    {
        Database = database;
        Username = username;
        Password = password;
        Hostname = hostname;        
    }
    
    public String Database;
    public String Username;
    public String Password;
    public String Hostname;
}